File: Database/src/main/java/sep3/database/Model/Topic.java
package sep3.database.Model;

import org.bson.types.ObjectId;

public class Topic {
    private String _id;
    private String name;

    public Topic(String _id, String name) {
        this._id = _id;
        this.name = name;
    }

    public Topic(String name) {

        this.name = name;
    }

    public String get_id() {
        return _id;
    }

    public void set_id(String _id) {
        this._id = _id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    @Override
    public String toString() {
        return "Topic{" +
                "_id=" + _id +
                ", name='" + name + '\'' +
                '}';
    }
}

File: Client/ChatClient/ChatClient/Models/Topic.cs
﻿namespace Application.Models
{
    public class Topic
    {
        public string _id { get; set; }
        public string name { get; set; }

        public Topic(string id, string name)
        {
            this.name = name;
            this._id = id;
        }

        public Topic(string name)
        {
            this.name = name;
        }

        public Topic()
        {
        }
    }
}